### ISSUE TITLE
Create Backup and Recovery Plan

### ISSUE THREAD

**Issue by ricetj** [department-of-veterans-affairs/va.gov-team#475]

**Comment by ricetj** [comment]
<a href="https://github.com/ShawnArnwine"><img src="https://avatars0.githubusercontent.com/u/1448049?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ShawnArnwine](https://github.com/ShawnArnwine)**
_Sunday Jul 24, 2016 at 18:37 GMT_

----

Mostly covered in our ATO-related SOPs and IRP/ISCP...but needs to be turned into checklists and properly assigned.

**Comment by ricetj** [comment]
<a href="https://github.com/dkhenry"><img src="https://avatars1.githubusercontent.com/u/489643?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [dkhenry](https://github.com/dkhenry)**
_Monday Jan 23, 2017 at 22:56 GMT_

----

We should also test out the recovery

# Things to Backup/Recover
- [ ] Infrastructure Generation
- [ ] Pet Instances ( CSR's ) 
- [ ] RDS
- [ ] Utility Instance Generation 
- [ ] Dev/Prod/Staging Instance Generation
- [ ] S3 Deployment

**Comment by ricetj** [comment]
(09/5/2017)
- [ ] Set up Matt and Wyatt with access to the test AWS govcloud accounts
- [ ] Document existing procedures and capabilities. Send to Matt and Wyatt for review

**Comment by ricetj** [comment]
(11/29/2017)
Pet instances (CSR's) basically just need to be snapshotted so we can restore them if needed.

**Comment by ricetj** [comment]
(01/17/2018)
blocked by DRP tabletop, will result in more specific tix to support this effort

**Comment by ricetj** [comment]
(05/16/2018)
Still relevant? 

I lean toward yes, but will leave this up to Wyatt.

I'm envisioning a brief breakdown of the types of state and systems we manage, and how we generally provide backups/restoration for those systems. Critical persistent state in RDS and whichever EBS volumes we rely on, and how we restore infrastructure with terraform/ansible. It would be useful to summarize what we'll restore from scratch vs what we need to back up.

**Comment by ricetj** [comment]
@wyattwalter Ported this ticket over. This looks like more of an ATO issue, but may come into the ops scope. Wanted to get your take on this. Would this be a shared effort with Ops and Product Support?

**Comment by andreahewitt-odd** [comment]
@ricetj Does this ticket need VSP work?

**Comment by ricetj** [comment]
@andreahewitt-odd This is another ticket that was ported over from Vets.gov repo. https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1542
This will most likely be an ATO ticket in the future but may touch a few teams. I think we need to wait till we have ATO up and going before taking this on.

**Comment by ricetj** [comment]
@ricetj to put this on top of the list of things to do.

**Comment by jhouse-solvd** [comment]
@ricetj Is this still active / pending / to-be-discovered?

**Comment by ricetj** [comment]
This is something that needs to be in the Q yes. Something the team needs to at the very least has an eye on for the future.

**Comment by rmtolmach** [comment]
We should create new initiatives to address this based on the state of current infrastructure. Leaving this until we can create new issues for that work.
